korangar/src/loaders/model/mod.rs

```diff
@@ -247,6 +247,9 @@ impl ModelLoader {
         };
 
         // TODO: Temporary check until we support more versions.
+        // TODO: The model operation to scale keyframe not implemented yet
+        // TODO: The model operation to translate keyframe not implemented yet
+        // TODO: The model operation to modify texture keyframe not implemented yet
```

Comment:
Please make sure that the comments use correct grammar (and end them on a `.` please)